blob/azureblob/azureblob.go

```diff
@@ -661,6 +668,7 @@ func (b *bucket) SignedURL(ctx context.Context, key string, opts *driver.SignedU
 		ContainerName: b.name,
 		BlobName:      srcBlobParts.BlobName,
 		Permissions:   perms.String(),
+		ContentType:   opts.ContentType,
```

Comment:
Should `opts.ContentType` be here? IIUC it can never be anything other than the empty string.

Reply:
Oops, good catch. A carryover from an earlier revision.